### ISSUE TITLE
Plugin configuration completion not working inside `<>`

### ISSUE THREAD

**Issue by mickaelistria** [eclipse/lemminx-maven#64]
Trying to complete inside the `<>` of 
```xml
<plugin>
	<groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-surefire-plugin</artifactId>
	<configuration>
		<>
	</configuration>
</plugin>
```
shows no useful results.
However, trying to complete outside of this element, just after `<configuration>` works.

**Comment by mickaelistria** [comment]
I looked at the logs, and it seems to be more an issue on client side filtering being too greedy, or maybe the completion items report wrong locations so they get filtered.

**Comment by AObuchow** [comment]
I was going to suggest maybe this was caused by https://github.com/eclipse/lemminx-maven/commit/e80b44bd476dffacacffd75e500932750c53062e although you have other suspicions which are more likely.

**Comment by mickaelistria** [comment]
Fix in #65
